generators/typegen/generator.go

```diff
@@ -126,15 +158,57 @@ func (g *Generator) generateTypesForPackage(s *schema.Schema, genConfig *config.
 			}
 
 			enumsForGen = append(enumsForGen, xxx)
+		case schema.KindScalar:
+			log.Warnf("scalar type: %+v", t)
+
+			// Default scalars to int
+			createAs := "int"
+			skipTypeCreate := false
+
+			for _, p := range pkgConfig.Types {
+				if p.Name == t.GetName() {
+					if p.CreateAs != "" {
+						createAs = p.CreateAs
+					}
+
+					if p.SkipTypeCreate {
+						skipTypeCreate = true
+					}
+				}
+			}
+
+			if !t.IsGoType() && !skipTypeCreate {
+				xxx := goScalar{
+					Description: t.GetDescription(),
+					Name:        t.GetName(),
+					Type:        createAs,
+				}
+
+				scalarsForGen = append(scalarsForGen, xxx)
```

Comment:
Can you not append a literal here? Not sure if it's more efficient, but more compact.

Reply:
I believe you can, I just think multi-line function calls are gross, and putting it on one line is also gross.